.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+# linting check of radis-app
+name: 🌱 CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  test:
+    name: 🧪 Test
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: Cache the dependencies
+        uses: c-hive/gha-yarn-cache@v2
```

Comment:
We don't need another action to setup dependency caching because the setup-node action will do this for us if we specify `cache: yarn`: https://github.com/actions/setup-node#caching-global-packages-data